src/mongoose.ts

```diff
@@ -152,8 +152,8 @@ export function htMongooseFactory(mongoose: any) {
     });
   }
 
```

Comment:
to be consistent with our semantics, shouldn't this be a "From"?
- [ ] name?

Reply:
done